modules/ROOT/pages/java/java-invoke-method.adoc

```diff
@@ -36,10 +44,24 @@ This example invokes a method that takes arguments:
 </java:invoke>
 ----
 
-IMPORTANT: When configuring the Method arguments in the `args` parameter, the keys of the map will identify which parameter to invoke. *To reference the parameters by name* (`firstName`, `age`, and so on), the Java class containing the Method or Constructor must be compiled using the https://docs.oracle.com/javase/tutorial/reflect/member/methodparameterreflection.html[`-parameters` compiler flag]. If the class was not compiled with this flag, then the same parameters should be refenced in the canonical way as `arg0`, `arg1`, and so on.
+For a simple example in Studio, see <<studio_example>>.
+
+IMPORTANT: When configuring method arguments in the `args` parameter, the keys
+of the DataWeave object identify which parameter to invoke.
+_To reference the parameters by name_ (for example, `price`), the Java class
+containing the method or constructor must be compiled using the
+https://docs.oracle.com/javase/tutorial/reflect/member/methodparameterreflection.html[`-parameters` compiler option].
+If the class is not compiled with this option, the same parameters must be
+referenced in the canonical way, as `arg0`, `arg1`, and so on. For the constructor
+to work with parameters that you reference by name, you also need to add the XML
+for the Maven compiler plugin to your `pom.xml` file for your Mule project.
+Otherwise, an error occurs.
+
+//EXAMPLE?
+Note that in Studio, the Java module provides DataSense to provide metadata
+associated with `<java:new/>` and both the output value and the input arguments.
+You can access the metadata
```

Comment:
Where is the rest of this sentence?

Reply:
that was removed some time ago, before this review. Strange that you're seeing it.